daemon/oci_windows.go

```diff
@@ -89,9 +90,15 @@ func (daemon *Daemon) createSpec(c *container.Container) (*libcontainerd.Spec, e
 
 	// s.Windows.LayerPaths
 	var layerPaths []string
-	if img.RootFS != nil && img.RootFS.Type == "layers+base" {
+	if img.RootFS != nil && img.RootFS.Type == image.TypeLayers || img.RootFS.Type == image.TypeLayersWithBase {
```

Comment:
Should this be `img.RootFS != nil && (img.RootFS.Type == image.TypeLayers || img.RoootFS.Type == image.TypeLayersWithBase)`?